src/app/results/results.component.html

```diff
@@ -33,22 +33,31 @@
                         {{item.pubDate|date:'fullDate'}}
                     </div>
                 </div>
-            </div>
+                <div class="video-result" *ngIf="Display('videos')">
+                    <div *ngFor="let item of items$|async" class="result">
+                        <video id="fetch-videos" width="320" height="240" controls>
```

Comment:
since these are just thumbnails ,replace video tag with image .there by there wont be any player